discord_bot/pyproject.toml

```diff
@@ -0,0 +1,94 @@
+[tool.poetry]
+name = "discord_bot"
+version = "0.1.0"
+description = ""
+authors = ["Piotr Krawiec <contact@piotrk.it>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.11,<3.12"
```

Comment:
we won't use python 3.10 or 3.12 in other files? 3.11 is the main version ?

Reply:
Currently - yes. Many tools explicitly DON'T support python 3.12 yet. So i have to set python to`<3.12`